packages/odyssey-storybook/src/components/odyssey-mui/Status/Status.stories.tsx

```diff
@@ -74,6 +80,18 @@ const storybookMeta: Meta<StatusProps> = {
 
 export default storybookMeta;
 
+const checkStatusStyles = async (
+  canvas: ReturnType<typeof within>,
+  testId: string,
+  expectedBackgroundColor: string,
+  expectedColor: string,
+) => {
+  const status = canvas.getByTestId(testId);
+  await expect(status).toHaveStyle(
+    `background-color: ${expectedBackgroundColor}`,
+  );
+  await expect(status).toHaveStyle(`color: ${expectedColor}`);
+};
```

Comment:
We shouldn't rely on test IDs because we're wanting to use accessibility selectors and want other people to use them as well.

We should be able to select by the "status" role. In fact, if we could, we should add that to `odysseyQuerySelector`. It will help consumers of Odyssey.